bundles/com.espressif.idf.ui/src/com/espressif/idf/ui/wizard/NewIDFProjectWizard.java

```diff
@@ -85,6 +91,18 @@ public void createControl(Composite parent)
 
 	}
 
+	@Override
+	public void dispose()
+	{
+		super.dispose();
+		IWorkbenchPage page = PlatformUI.getWorkbench().getActiveWorkbenchWindow().getActivePage();
+		IViewPart viewPart = page.findView("org.eclipse.ui.navigator.ProjectExplorer");
+		ISelectionProvider selProvider = viewPart.getSite().getSelectionProvider();
+		String projectName = mainPage.getProjectName();
+		selProvider.setSelection(
+				new StructuredSelection(ResourcesPlugin.getWorkspace().getRoot().getProject(projectName)));
+	}
```

Comment:
Could you move this code to performFinish()..You can override this method. Since we are performing this operating after the project finish, this is a good place to keep.

```
@Override
	public boolean performFinish() {
		boolean performFinish = super.performFinish();
		if (performFinish) {
                    // your code
              }
return performFinish;
}
```

Reply:
@kolipakakondal logic was moved to perfomFinish()
https://github.com/espressif/idf-eclipse-plugin/blob/b12bd525186fac1a1603df0b7a6071b6169ec5de/bundles/com.espressif.idf.ui/src/com/espressif/idf/ui/wizard/NewIDFProjectWizard.java#L94-L111